spikeinterface/core/waveform_extractor.py

```diff
@@ -92,14 +106,36 @@ def __repr__(self):
         return txt
 
     @classmethod
-    def load_from_folder(cls, folder):
+    def load_from_folder(cls, folder, with_recording=True):
         folder = Path(folder)
-        assert folder.is_dir(), f'This folder does not exists {folder}'
-        recording = load_extractor(folder / 'recording.json',
-                                   base_folder=folder)
+        assert folder.is_dir(), f'This waveform folder does not exists {folder}'
+
+        if not with_recording:
+            # load
+            recording = None
+            rec_attributes_file = folder / 'recording_info' / 'recording_attributes.json'
+            if not rec_attributes_file.exists():
+                raise ValueError('This WaveformExtractor folder was created with oled version of spikeinterface'
+                                 'you cannot use the mode with_recording=False')
+            with open(rec_attributes_file, 'r') as f:
+                rec_attributes = json.load(f)
+            # the probe is handle ouside the main json
+            probegroup_file = folder / 'recording_info' / 'probegroup.json'
+            if probegroup_file.is_file():
+                rec_attributes['probegroup'] = probeinterface.read_probeinterface(probegroup_file)
+            else:
+                rec_attributes['probegroup'] = None
+        else:
+            try:
+                recording = load_extractor(folder / 'recording.json',
+                                        base_folder=folder)
+                rec_attributes = None
+            except:
+                raise Exception("The recording could not be loaded. You can use the `without_recording=True` argument")
```

Comment:
I think this should be "You can use `with_recording=False`" ?

Reply:
Yep, thanks!